src/main/java/org/seattlevoluntech/storage/ProjectsRepository.java

```diff
@@ -8,12 +8,14 @@
 import org.springframework.stereotype.Component;
 
 import java.util.List;
-import java.util.Optional;
 
 @Component
 public interface ProjectsRepository extends CrudRepository<Project, Long> {
 
     @Query("SELECT p FROM Project AS p WHERE p.status = :status")
     List<Project> findProjectByStatus(@Param("status") String status);
 
+    @Query(value = "SELECT * FROM Project ORDER BY creation_date DESC LIMIT :numberOfProjectsToDisplay", nativeQuery = true)
```

Comment:
Might want to only grab the latest projects by passing the status as a parameter? Then the frontend can get the most recent active projects

Reply:
Yea, the issue descriptions aren't fully described with every detail since we don't have an actual PM for the project. They were created on a best effort basis. The experience that is desired however, is the ability for a user to be able to see active projects on the home page. It doesn't make a lot of sense to see inactive ones.

Reply:
I've updated to enable ?status=<STATUS> param with default of "open" for project status.